### PR TITLE
virsh.emulatorpin: Restore VM xml incase of failure for vcpu placement

### PR DESCRIPTION
**Pull request by sathnaga**
Before this patch if VM fails to start after a vcpu placement
changes in xml, it leaves the buggy xml in the system and results in
further tests failing, this patch will restore the backupxml incase
of failout.

### CODE REVIEW

File: libvirt/tests/src/virsh_cmd/domain/virsh_emulatorpin.py [hunk 1]
```diff
@@ -194,7 +194,14 @@ def run(test, params, env):
         vm.destroy()
         vmxml.placement = emulatorpin_placement
         vmxml.sync()
-        vm.start()
+        try:
+            vm.start()
+        except:
```
Comment (balamuruhans) [inline libvirt/tests/src/virsh_cmd/domain/virsh_emulatorpin.py]: it is better to except for specific error "LibvirtXMLError"

**Comment by sathnaga** [comment]
@balamuruhans Thanks for the comment, have fixed :-)

**Comment by sathnaga** [comment]
Fixed CI failure

**Comment by balamuruhans** [comment]
ACK

**Comment by waynesun09** [comment]
ACK

**Pull request merged**